Sources/Internal/CoreStoreFetchRequest+CoreStore.swift

```diff
@@ -36,6 +36,6 @@ internal extension CoreStoreFetchRequest {
     @nonobjc
     internal func dynamicCast<U: NSFetchRequestResult>() -> NSFetchRequest<U> {
         
-        return unsafeBitCast(self, to: NSFetchRequest<U>.self)
+        return unsafeDowncast(self, to: NSFetchRequest<U>.self)
```

Comment:
By the way, `ResultType` is covariant. So, is `unsafeDowncast` really going to fail in this case?